src/runcrate/cli.py

```diff
@@ -56,15 +64,30 @@ def cli():
     type=click.Path(exists=True, dir_okay=False, readable=True, path_type=Path),
     help="path to a README file (should be README.md in Markdown format)",
 )
-def convert(root, output, license, workflow_name, readme):
+def convert(root, converter, output, license, workflow_name, readme):
     """\
     Convert a CWLProv RO bundle into a Workflow Run RO-Crate.
 
     RO_DIR: top-level directory of the CWLProv RO
     """
+
     if not output:
         output = Path(f"{root.name}.crate.zip")
-    builder = ProvCrateBuilder(root, workflow_name, license, readme)
+
+    if converter not in CONVERTERS:
```

Comment:
Not needed, this is handled by click

Reply:
Sounds reasonable to me! I was trying to cover off programmatic use but that probably needs more thought!